habitat-baselines/habitat_baselines/rl/hrl/hrl_rollout_storage.py

```diff
@@ -96,9 +98,9 @@ def insert(
 
```

Comment:
Add here 
```
self._last_should_inserts = should_inserts
```
Before returning

Reply:
See this PR
https://github.com/facebookresearch/habitat-lab/commit/ea2845e132f58e7769b956464ea28de1235a9eee